AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoModel.swift

```diff
@@ -242,16 +240,30 @@ class VideoModel: NSObject {
     }
 
     func getPreviousRemoteVideoPage() {
+        var removedList: [RemoteVideoSource] = []
         for remoteVideoTileState in remoteVideoStatesInCurrentPage {
-            audioVideoFacade.pauseRemoteVideoTile(tileId: remoteVideoTileState.0)
+            //audioVideoFacade.pauseRemoteVideoTile(tileId: remoteVideoTileState.0)
+            for (key, _) in remoteVideoSourceConfigurations {
```

Comment:
sadddd, would be much easier if `RemoteVideoSource` was in `RemoteVideoTileState`...

Reply:
still sad lol..this function could be like
```
audioVideoFacade.updateVideoSourceSubscriptions(addedOrUpdated: [:], removed: remoteVideoStatesInCurrentPage.map { it.source })
```
. In a perfect world we would combine the call to add with the one to remove. Is that possible?